MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
+  "people": {
+    "gracehopper": {
```

Comment:
Ah yeah, that's holdover thinking from roles. 👍 on people being an array. I just pushed a commit with people as an array.